From 8570aa372d00aa6410605b97a78614c61d74c4b3 Mon Sep 17 00:00:00 2001
From: Michael Cronenworth <mike@cchtml.com>
Date: Fri, 1 Jul 2016 16:31:11 -0500
Subject: [PATCH] winspool.drv: Support cups 2.2 header update

Signed-off-by: Michael Cronenworth <mike@cchtml.com>
---
 dlls/winspool.drv/info.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dlls/winspool.drv/info.c b/dlls/winspool.drv/info.c
index 3ba94cd..5468066 100644
--- a/dlls/winspool.drv/info.c
+++ b/dlls/winspool.drv/info.c
@@ -43,6 +43,9 @@
 #include <signal.h>
 #ifdef HAVE_CUPS_CUPS_H
 # include <cups/cups.h>
+# if CUPS_VERSION_MAJOR > 2 || (CUPS_VERSION_MAJOR == 2 && CUPS_VERSION_MINOR > 1)
+#  include <cups/ppd.h>
+# endif
 #endif
 
 #ifdef HAVE_APPLICATIONSERVICES_APPLICATIONSERVICES_H
-- 
2.5.5

