--- a/patches/patchinstall.sh	2015-10-31 18:16:04.000000000 -0500
+++ b/patches/patchinstall.sh	2015-11-04 23:26:11.214562133 -0600
@@ -2371,12 +2371,12 @@
 # | Modified files:
 # |   *	configure.ac
 # |
-if test "$enable_configure_Absolute_RPATH" -eq 1; then
-	patch_apply configure-Absolute_RPATH/0001-configure-Also-add-the-absolute-RPATH-when-linking-a.patch
-	(
-		echo '+    { "Sebastian Lackner", "configure: Also add the absolute RPATH when linking against libwine.", 1 },';
-	) >> "$patchlist"
-fi
+#if test "$enable_configure_Absolute_RPATH" -eq 1; then
+#	patch_apply configure-Absolute_RPATH/0001-configure-Also-add-the-absolute-RPATH-when-linking-a.patch
+#	(
+#		echo '+    { "Sebastian Lackner", "configure: Also add the absolute RPATH when linking against libwine.", 1 },';
+#	) >> "$patchlist"
+#fi
 
 # Patchset crypt32-CMS_Certificates
 # |
@@ -4477,14 +4477,14 @@
 # | 	dlls/ntoskrnl.exe/tests/driver.sys/util.h, dlls/ntoskrnl.exe/tests/ntoskrnl.c, include/wine/test.h,
 # | 	tools/make_makefiles, tools/makedep.c
 # |
-if test "$enable_ntoskrnl_DriverTest" -eq 1; then
-	patch_apply ntoskrnl-DriverTest/0001-ntoskrnl.exe-tests-Add-initial-driver-testing-framew.patch
-	patch_apply ntoskrnl-DriverTest/0002-ntoskrnl.exe-tests-Add-kernel-compliant-test-functio.patch
-	(
-		echo '+    { "Sebastian Lackner", "ntoskrnl.exe/tests: Add initial driver testing framework and corresponding changes to Makefile system.", 2 },';
-		echo '+    { "Michael Müller", "ntoskrnl.exe/tests: Add kernel compliant test functions.", 1 },';
-	) >> "$patchlist"
-fi
+#if test "$enable_ntoskrnl_DriverTest" -eq 1; then
+#	patch_apply ntoskrnl-DriverTest/0001-ntoskrnl.exe-tests-Add-initial-driver-testing-framew.patch
+#	patch_apply ntoskrnl-DriverTest/0002-ntoskrnl.exe-tests-Add-kernel-compliant-test-functio.patch
+#	(
+#		echo '+    { "Sebastian Lackner", "ntoskrnl.exe/tests: Add initial driver testing framework and corresponding changes to Makefile system.", 2 },';
+#		echo '+    { "Michael Müller", "ntoskrnl.exe/tests: Add kernel compliant test functions.", 1 },';
+#	) >> "$patchlist"
+#fi
 
 # Patchset ntoskrnl-Stubs
 # |
